.github/workflows/main.yaml

```
@@ -61,6 +61,7 @@ jobs:
           - sphinx.sh
           - black.sh
           - blue.sh
+          - smoke.sh
```

```suggestion
          - coc_citizen.sh
```

the name here should be coc_citizen.sh